README.md

```diff
@@ -151,7 +151,7 @@ Be sure that GraalVM is installed by running the following command, otherwise yo
 User: `Deploy in OpenShift the module http-minimum compiled with my local GraalVM in order to build my application`
 
 ```shell
-mvn clean verify -Popenshift -Dall-modules -Dquarkus.package.type=native -pl http/http-minimum
+mvn clean verify -Popenshift -Dall-modules -Dquarkus.package.jar.type=native -pl http/http-minimum
```

Comment:
I think this is wrong, `fast-jar,legacy-jar,uber-jar,mutable-jar` should be the allowed only ones @michalvavrik

Reply:
You are right, luckily it's just a README, I'll create a fix.